src/test/regress/sql/multi_partitioning_utils.sql

```diff
@@ -0,0 +1,218 @@
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1620000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1620000;
```

Comment:
Let's not set these if we don't need them. We don't even use `pg_dist_jobid_seq` anymore.